src/configure/helper/LocalGitRepoHelper.ts

```diff
@@ -46,63 +44,23 @@ export class LocalGitRepoHelper {
         return deferred.promise;
     }
 
-    public async getGitRepoDetails(repositoryPath: string): Promise<GitRepositoryParameters> {
+    public async getGitRepositoryDetails(): Promise<GitRepositoryDetails> {
```

Comment:
should this be called getGitBranchDetails

Reply:
yes sounds better.